app/scripts/controllers/createRoute.js

```diff
@@ -17,6 +17,7 @@ angular.module('openshiftConsole')
                        AuthorizationService,
                        DataService,
                        Navigate,
+                       NotificationsService,
                        ProjectsService,
                        keyValueEditorUtils) {
     $scope.alerts = {};
```

Comment:
You can remove `$scope.alerts` since it's no longer used and also remove the `alerts` element from create-route.html